command/recorderd/subscriber.go

```diff
@@ -95,13 +105,20 @@ func Subscribe(i int, connectTo string, v6 bool, serverPublicKey []byte, publicK
 			err = json.Unmarshal([]byte(data), &item)
 			log.Infof("received : %v", item)
 
+			// prevent queuing outdated request
+			if !proofer.isWorking() {
```

Comment:
Clean the ZeroMQ buffer instead?

Reply:
Just check `zmq4.go`, it seems there's no api to clear queue.

Reply:
But recheck the code, I think this block can be moved to line 101, which reduce cpu calculation of `json.unmarshal`